module3/projects/consultancy/ideation.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: page
+title: Consultancy - Ideation
+type: project
+---
+_[Back to Consultancy Home](./index)_ 
+## Timeline
+
+* End of Week 2: Students begin brainstorming project ideas individually using the following guidelines.
+* Week 3: Students submit project pitch summaries via form
+* End of Week 3: Students deliver pitches to cohort and vote on preferred project
+
+### Project Template
+
+Fill out the following template for your project idea.
+
+```markdown
+### [Project Name]
+
+### Pitch
+
+2 - 3 sentences that explains what the application will do.
+
+### End User
+
+* Who is this app target towards? Who is your user?
+
+### Problem
+
+* What problem is your app trying to solve? Why is your app different from any of the other 'similar' apps that are out there.
+
+### Features
+
+* What features will your app include?
+
+### Equity Anaylsis
+* Features or tools you plan on building into the project to be sure it promotes inclusion. Questions to consider (should include one or more of these): 
+   * Are any of the following marginalized identity groups (or others) likely to experience harm or exclusion as a result of this project? 
+      * E.g., immigrants, people of color, people with (dis)/Abilities, justice impacted (formerly incarcerated) people, elderly people, women, gender expansive people, people with limited incomes, etc.
+   * Which factors may impact any individual or group’s access to the proposed project?
+   * If you were given more time to design this product, how would you want to work with your intended users to ensure relevance?
+   * What happens if someone uses this product with bad/evil intent in mind?
+
+### Integrations
+
+* Which API(s) are you thinking of using? If there are no free APIs, how else could you acquire external data?
+* Are any of the possible BE stretch techs particularly well-suited for this project?
+```
+
+### APIs
+
+Your application **must make good use of at least one external dataset or API**. Some examples include:
+
+#### Government Data
```

Comment:
I love that we include a list of examples here. Finding public APIs can feel daunting to them!